### PR TITLE
尝试修复 #97 中提出的顶级容器滚动，却没有触发 container[overflow] 容器内元素检查 visible 的情况

### PR DESCRIPTION
**Pull request by zemzheng**
RT

### CODE REVIEW

**Comment by zemzheng** [comment]
顺带解决 #84 的 overflow 问题

File: src/utils/scrollParent.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ export default (node) => {
       continue;
     }
 
-    if (overflowRegex.test(overflow) && overflowRegex.test(overflowX) && overflowRegex.test(overflowY)) {
+    // https://github.com/jasonslyvia/react-lazyload/issues/84#issuecomment-308607651
+    if ( overflowRegex.test(overflowX) && overflowRegex.test(overflowY)) {
```
Comment (davydog187) [inline src/utils/scrollParent.js]: See my issue #104 I believe this should address the issue for chrome as well.

**Comment by gajus** [comment]
Any chance on this getting merged?

File: test/specs/lazyload.spec.js [hunk 2]
```diff
@@ -190,6 +190,30 @@ describe('LazyLoad', () => {
         done();
       }, 500);
     });
+
+  it('Item inside overflow-container should be visitble when the document has scroll to show it', (done) => {
```
Comment (gajus) [inline test/specs/lazyload.spec.js]: `s/visitble/visible`

**Comment by zemzheng** [comment]
For my own project, i merged it to react-lazyload2

**Pull request closed without merging**